beets/ui/commands.py

```diff
@@ -873,102 +871,66 @@ def import_files(lib, paths, query):
     plugins.send('import', lib=lib, paths=paths)
 
 
-def import_func(lib, opts, args):
+# TODO: add 'imp' and 'im' aliases
```

Comment:
Ah yes, we'll need to do aliases—looks like we'll need to do more with that MultiCommand? http://click.pocoo.org/4/advanced/#command-aliases